pubsub/rabbitmq/rabbitmq.go

```diff
@@ -48,17 +49,19 @@ const (
 	defaultHeartbeat        = 10 * time.Second
 	defaultLocale           = "en_US"
 
-	argQueueMode              = "x-queue-mode"
-	argMaxLength              = "x-max-length"
-	argMaxLengthBytes         = "x-max-length-bytes"
-	argDeadLetterExchange     = "x-dead-letter-exchange"
-	argMaxPriority            = "x-max-priority"
-	propertyClientName        = "connection_name"
-	queueModeLazy             = "lazy"
-	reqMetadataRoutingKey     = "routingKey"
-	reqMetadataQueueTypeKey   = "queueType" // at the moment, only supporting classic and quorum queues
-	reqMetadataMaxLenKey      = "maxLen"
-	reqMetadataMaxLenBytesKey = "maxLenBytes"
+	argQueueMode                       = "x-queue-mode"
+	argMaxLength                       = "x-max-length"
+	argMaxLengthBytes                  = "x-max-length-bytes"
+	argDeadLetterExchange              = "x-dead-letter-exchange"
+	argMaxPriority                     = "x-max-priority"
+	argSingleActiveConsumer            = "x-single-active-consumer"
```

Comment:
Please open another PR to update https://github.com/dapr/components-contrib/blob/main/pubsub/rabbitmq/metadata.yaml
CC: @yaron2

Reply:
@ItalyPaleAle It's not clear to me what should be updated in the `metadata.yml` file. This file seems to describe metadata for the rabbitmq pubsub **Component** while the `singleActiveConsumer` parameter is set at the **Subscription** level (like `queueName`, `routingKey`, ...)

Reply:
I see, I misunderstood this PR, thanks for clarifying.